tools/github-repo-configurator/README.md

```diff
@@ -0,0 +1,37 @@
+# GitHub Repo Configurator
+GitHub Repo Configurator configures a GitHub repository according to a defined repository manifest.
+
+## Building the tool
+You can easily build the tool with the .NET Core commands:
+```shell
+dotnet build .\src
+```
+
+## Running the tool
+Run the tool
+```shell
+Arcus.Tools.Repository.App.exe
```

Comment:
Does `dotnet build` produce an `.exe`? I would've thought it was a `.dll`, if it was a .NET Core project.

Reply:
Good point, I actually need to check this - Assumed that this was the case which is wrong.